test/e2e/route_metrics_test.go

```diff
@@ -247,7 +294,14 @@ func testRouteMetricsControllerLabelSelector(t *testing.T, testRS, testNS bool)
 		// Update the RouteSelector of the Ingress Controller so that the Route gets admitted again.
 		ic.Spec.RouteSelector = &metav1.LabelSelector{
 			MatchLabels: map[string]string{
-				"type": correctLabel,
+				"label": correctLabel,
+			},
+			MatchExpressions: []metav1.LabelSelectorRequirement{
+				{
+					Key:      "expression",
+					Operator: metav1.LabelSelectorOpIn,
+					Values:   []string{incorrectLabel},
```

Comment:
A copy-paste issue I guess.
```suggestion
					Values:   []string{correctLabel},
```

Reply:
Ah thanks!